lib/App/DuckPAN/DDG.pm

```diff
@@ -74,11 +92,21 @@ sub get_blocks_from_current_dir {
 		if (my $ia = $self->app->get_ia_by_name($arg)) {
 			$class = $ia->{perl_module};
 		}
+		elsif ($arg =~ /_/) {
```

Comment:
Good catch 👍 

I will check if it matches `/^[a-z0-9_]+$/` which resembles a lowercased ID.

If there are uppercase letters we can assume it's a Perl Module Name.